### ISSUE TITLE
[FEATURE REQUEST] copy individual image seeds?

### ISSUE THREAD

**Issue by altryne** [hlky/stable-diffusion-webui#195]
**Is your feature request related to a problem? Please describe.**
Not sure if this is possible, but I remember if you didn't specify seed, SD discord would spit out seeds for every sample, so you can hone a specific one. 
We currently show only 1 seed, but does each sample have its own seed? 

@hlky do you know?

If every sample has it's own seed, we can somehow expose this in the ui when you select a thumbnail (given the JS code that I added

**Comment by hlky** [comment]
>does each sample have its own seed?

Yes seed should increment and files are saved with the incrementing seed

With that JS code yes it should be trivial to implement so that the copy and paste text with generation settings updates with the selected image's seed

**Comment by altryne** [comment]
Oh it's literally incrementing? 
if I ask for 4 samples and provide a seed like 1234
it'll be 
1234, 1235,1236,1237?

**Comment by hlky** [comment]
Yes, images made in the next 'batch' (controlled by batch count, originally 'n_iter') have the seed incremented, images made in the same  batch ('batch size' previously 'n_samples') should have the same seed afaik, or at least that should be the intended behavior.
